README.rst

```diff
@@ -713,6 +713,12 @@ Choose from the list of available rules:
 
   Fix phpdoc inline tags, make inheritdoc always inline.
 
+  Configuration options:
+
+  - ``psr5`` (``bool``): whether the fixer should be PSR-5 compliant (uses
```

Comment:
hmm I wonder, IIRC the only reason why we always put the tag with braces ATM is because SF needed that as a work around. Maybe worth checking if this is still the case.